.github/workflows/php-tests.yml

```
@@ -30,21 +30,26 @@ jobs:
         id: get-wordpress-versions
         uses: ./.github/actions/get-wordpress-versions
         with:
-          version-count: 3
+          version-count: 2
 
   test:
     runs-on: ubuntu-22.04
     needs: [ get-woocommerce-versions, get-wordpress-versions ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)